ITEventsTests/DateFormatterServiceTests.swift

```diff
@@ -0,0 +1,53 @@
+import XCTest
+@testable import ITEvents
+
+class DateFormatterServiceTests: XCTestCase {
+    var dateFormatterService: DateFormatterService!
+    var dateIntervalSameDate: DateInterval!
+    var dateIntervalDifferentDates: DateInterval!
+    
+    override func setUp() {
+        super.setUp()
+        
+        dateFormatterService = DateFormatterService()
+
+        let dateFormatter: DateFormatter = {
+            let formatter = DateFormatter()
+            formatter.dateFormat = "yyyy-MM-dd HH:mm:ss"
+            return formatter
+        }()
+        
+        let startDate = "2018-01-18 19:00:00"
+        let endDate = "2018-01-18 22:00:00"
+        let endDifferentDate = "2018-01-19 22:00:00"
+        
+        dateIntervalSameDate = DateInterval(start: dateFormatter.date(from: startDate)! ,
+                                            end: dateFormatter.date(from: endDate)!)
+        dateIntervalDifferentDates = DateInterval(start: dateFormatter.date(from: startDate)!,
+                                                  end: dateFormatter.date(from: endDifferentDate)!)
```

Comment:
I'd do the following:
- removed extra variables (```startDate```, ```endDate```, ```endDifferentDate```)  as they only making a noise here 
- renamed the DateInterval variables to be more explicit

```swift     
sameDayDateInterval = DateInterval(start: dateFormatter.date(from:  "2018-01-18 19:00:00")! ,
                                   end: dateFormatter.date(from: "2018-01-18 22:00:00")!)
differentDaysDateInterval = DateInterval(start: dateFormatter.date(from: "2018-01-18 19:00:00")!,
                                         end: dateFormatter.date(from: "2018-01-19 22:00:00")!)

```